pkg/apis/ceph.rook.io/v1/object.go

```diff
@@ -76,6 +76,9 @@ func ValidateObjectSpec(gs *CephObjectStore) error {
 	if gs.Namespace == "" {
 		return errors.New("missing namespace")
 	}
+	if len(gs.Name) > 38 {
+		return errors.New("object store name cannot be bigger than 38 characters")
+	}
```

Comment:
```suggestion
	if len(gs.Name) > 38 {
		return errors.New("object store name cannot be bigger than 38 characters")
	}
```

Also, as I mentioned let's move this block to L63 inside `ValidateCluster` as this method will trigger during the update also and will cause errors because earlier it was >38 and now webhook will validate with <38 and it will through an error.

@travisn right?

Reply:
This method is called by `ValidateCreate()` on L65, so why do we need to move it? I didn't follow.